### ISSUE TITLE
Graph2ToCoPo model

### ISSUE THREAD

**Issue by nashid** [google-research/plur#2]
Thanks for the artefact. I have a couple of questions about this artefact. After reading  the paper:

(1) One major contribution the paper claims to provide is the open-sourced framework of PLUR that others can use. I am wondering whether you are planning to share the model and training scripts?

(2) I am particularly interested to run your model for Hoppity. Is there a possibility you can share the model even if that is partial?  

I am wondering whether you would share the models and steps for the training code?

**Comment by nashid** [comment]
Wondering when the model would be open-sourced? Is there a tentative timeline?

**Comment by smoitra-g** [comment]
Hi @nashid , thanks for your interest in the PLUR project and apologies for the delay in replying. We are close to releasing the the modeling and training code. Maybe a couple of weeks if not days.

**Comment by nashid** [comment]
@smoitra-g this is fantastic. I am wondering whether will you be sharing the model itself?

**Comment by smoitra-g** [comment]
Yes the release will have the {GGNN,Transformer,GREAT}2Tocopo models.

**Comment by nashid** [comment]
This is fantastic. Looking forward to it 👍

**Comment by nashid** [comment]
@smoitra-g I see you have pushed some code for the model. Would be helpful to know once all the changes are pushed in.

**Comment by smoitra87** [comment]
Hey @nashid . It should be code complete. Give it a try!! You'll be one of the alpha testers.
